### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by Void48**
also changed the Discord gh badge color to the new blurple: https://discord.com/branding

### CODE REVIEW

**Comment by night** [comment]
Thanks for the PR! The logo doesn't appear to be transparent though, can that be updated? We probably also want to store it in the repo versus using a GitHub hosted URL that the repo does not control.

**Comment by Void48** [comment]
> Thanks for the PR! The logo doesn't appear to be transparent though, can that be updated? We probably also want to store it in the repo versus using a GitHub hosted URL that the repo does not control.

I will look into it tomorrow 👍

**Comment by Void48** [comment]
Should be good now @night
when you merge it it should be displayed in the README using the path to the file

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# BetterTTV
+# <img src="https://raw.githubusercontent.com/night/betterttv/master/.github/bttvlogotransparent.gif" height="50"> BetterTTV
```
Comment (Felanbird) [inline README.md:1]:
```suggestion
# <img src=".github/bttvlogotransparent.gif" height="50"> BetterTTV
```
This works as a cleaner approach, though I'm unsure if `.github` is the best place to drop this

**Comment by night** [comment]
Thanks for the update. Is it possible to get the GIF's edges to be smoother? Maybe due to its size the edges look pretty jagged, and making it smaller may help. If not, I might propose using the logo without animation so we can achieve higher quality.

**Comment by Void48** [comment]
> Thanks for the update. Is it possible to get the GIF's edges to be smoother? Maybe due to its size the edges look pretty jagged, and making it smaller may help. If not, I might propose using the logo without animation so we can achieve higher quality.

Dealing with the .gif format is pretty annoying. It's definitely possible but I don't think I want to spend any more time on this😕
I tried enabling light mode on GitHub to see if the edges are that noticeable but I think it's honestly fine if it's just for the README

File: README.md (hunk 1 above)
Comment (Void48) [reply README.md]: thanks by the way :bowtie:

**Pull request merged**